compiler/luci/import/src/Importer.cpp

```diff
@@ -102,16 +102,18 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
     // Data type
     graph_input->dtype(input_node->dtype());
 
-    assert(tensor.shape_signature.size() == 0 ||
-           tensor.shape_signature.size() == tensor.shape.size());
+    const auto tensor_shape_signature = luci::wrap(tensor->shape_signature());
+    const auto tensor_shape = luci::wrap(tensor->shape());
+    assert(tensor_shape_signature.size() == 0 ||
+           tensor_shape_signature.size() == tensor_shape.size());
 
     // Shape of GraphInput
     auto input_shape = std::make_unique<loco::TensorShape>();
-    const std::vector<int32_t> &input_dims = tensor.shape; // in NHWC
+    const auto input_dims = tensor_shape; // in NHWC
```

Comment:
will this make a copy? if so, how about using `const auto &input_dims` ?

Reply:
This line makes a copy of `VectorWrapper` struct, but I would like to use `&` here. Thank you!

```suggestion
    const auto &input_dims = tensor_shape; // in NHWC
```